demo-ui/app/uk/gov/ons/addressIndex/demoui/views/multiMatch.scala.html

```diff
@@ -137,6 +137,12 @@ <h3 class="panel-title">@messages("multi.sfatext")</h3>
             </div>
             <div id="search-loading"></div>
             @results.map { bulkResp =>
+                    <div>
+                        <b>Number of addresses searched:</b> @{MatchTypeHelper.countSearched(bulkResp.bulkAddresses)} <br>
+                        <b>Single matches:</b> @{MatchTypeHelper.countSingles(bulkResp.bulkAddresses)}<br>
+                        <b>Multiple matches:</b> @{MatchTypeHelper.countMultiples(bulkResp.bulkAddresses)}<br>
+                        <b>No match:</b> @{MatchTypeHelper.countEmpty(bulkResp.bulkAddresses)}
+                    </div>
```

Comment:
The <b> tag is frowned on for HTML 5 see https://www.w3schools.com/tags/tag_b.asp
Should really use messages for "Number of addresses searched:" etc.